.github/workflows/list_components.yml

```diff
@@ -31,59 +31,95 @@ jobs:
         shell: bash
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
     - name: Intel® oneAPI Base Toolkit
-      run: scripts/list_components_windows.bat $WINDOWS_BASEKIT_URL
+      run: scripts/list_components_windows.bat $WINDOWS_BASEKIT_URL ${{ github.workflow }}_${{ github.job }}_basekit
```

Comment:
Suggestion: Put the generated files in build/list_components directory and add a .txt suffix.

Add build to the .gitignore so they will not be accidentally committed.

Below, you added a bunch of empty files, not sure why you did that.

Reply:
Thanks. I will add a .txt suffix and move them in a subdirectory,
Structurally, those files are part the source for the doc rather than the resulting doc. I think they belong in the source directory.

There were 2 reasons I committed the empty files:
1) prevent doc8 check failure: 
As part of D000, doc8 performs a link-check on index.rst file. If a file referenced by an "include" directive is missing, doc8 fails. I haven't found a way to disable the link-check separately from other RST syntax checks.
2) Make it possible to build the doc locally, with only the CI-generated parts being empty.
An alternative is to generate the empty files as part of the local build, but it would require different code for local and CI builds - currently it's simple "make html".

What if I add placeholder data in those files instead of leaving them empty? E.g: "this content will be generated by CI job".

Reply:
I understand now. It is difficult to generate the files without the CI. Moving into a separate directory and placeholder data seems like the best solution.

Reply:
I have made the changes in the last commit.